pagespeed/envoy/cluster.yaml

```diff
@@ -0,0 +1,41 @@
+static_resources:
```

Comment:
Instead of this file, can we use the api's to set up the cluster?
for an example, see https://github.com/envoyproxy/nighthawk/blob/master/source/client/process_impl.cc#L232